mesonbuild/mcompile.py

```diff
@@ -83,14 +120,22 @@ def run(options: 'argparse.Namespace') -> int:
 
         cmd = [runner, '-C', bdir.as_posix()]
 
+        if options.targets and options.clean:
+            raise MesonException('`target` and `clean` can\'t be used simultaneously')
+        if options.targets:
+            with (bdir/'meson-info'/'intro-targets.json').open() as f:
```

Comment:
`/` is an operator, and we always use spaces around operators.

Reply:
Fixed